pyiode/objects/equation.pyx

```diff
@@ -276,6 +292,7 @@ cdef class Equation:
         """
         return [coeff.decode() for coeff in self.c_equation.get_coefficients_list(create_if_not_exit)]
 
+    # same comments as for coefficients
```

Comment:
Implemented a *variables* property + a hidden method _get_and_create_variables() in PR #580